zipkin-server/README.md

```diff
@@ -21,13 +21,17 @@ Once you've started, browse to http://your_host:9411 to find traces!
 The following endpoints are defined under the base url http://your_host:9411
 * / - [UI](../zipkin-ui)
 * /config.json - [Configuration for the UI](#configuration-for-the-ui)
-* /api/v1 - [Api](http://zipkin.io/zipkin-api/#/)
+* /api/v2 - [Api](https://zipkin.io/zipkin-api/#/)
 * /health - Returns 200 status if OK
 * /info - Provides the version of the running instance
 * /metrics - Includes collector metrics broken down by transport type
 
 There are more [built-in endpoints](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html) provided by Spring Boot, such as `/metrics`. To comprehensively list endpoints, `GET /mappings`.
 
+The [legacy /api/v1 Api](https://zipkin.io/zipkin-api/#/) is still supported. Backends are decoupled from the
+HTTP api via data conversion. This means you can still accept legacy data on new backends and visa versa. Enter
```

Comment:
So is it possible to deploy the newest Zipkin with the old data format on the backend?

Reply:
yeap!